lit/docs/guides/container-images.lit

```diff
@@ -0,0 +1,318 @@
+  assumes you understand how to build container images with
+  \link{Dockerfile's}{https://docs.docker.com/engine/reference/builder/} and
+  publish to \link{Docker Hub}{https://hub.docker.com/} or another image
+  registry using the docker cli.
+
+  \aside{
+  This is one way of building and pushing images. There are many other ways to
+  accomplish this same task in Concourse.
+  }
+
+  First we need a Dockerfile. You can store this in your own repo or reference
+  the \link{github.com/concourse/examples}{https://github.com/concourse/examples}
+  repo. The rest of this post assumes you use the examples repo. All files in
+  this blog post can be found in the examples repo.
+
+  \inline-header{The Dockerfile}
+
+  \titled-codeblock{Dockerfile}{dockerfile}{{
+  FROM busybox
+
+  RUN echo "I'm simple!"
+  COPY ./stranger /stranger
+  RUN cat /stranger
+  }}
+
+  \inline-header{Defining Pipeline Resources}
+
+  Now we can start building out our pipeline. Let's declare our \reference{resources} first.
+  We will need one resource to pull in the repo where our Dockerfile is located,
+  and a second resource pointing to where we want to push the built container
+  image to.
+
+  \italic{
+  There are some \reference{variables} in this file that we will fill out when
+  setting the pipeline.
+  }
+
+  \titled-codeblock{build-push.yml}{yaml}{{
+  resources:
+  # The repo with our Dockerfile
+  - name: concourse-examples
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/concourse/examples.git
+      branch: master
+
+  # Where we will push the image to
+  - name: simple-image
+    type: registry-image
+    icon: docker
+    source:
+      repository: ((image-repo-name))/simple-image
+      username: ((registry-username))
+      password: ((registry-password))
+  }}
+
+  \inline-header{Create the Job}
+
+  Next we will create a \reference{jobs}{job} that will build and push our container image.
+
+  To build the job we will need to pull in the repo where the \code{Dockerfile} is.
+
+  \titled-codeblock{build-push.yml}{yaml}{{
+  resources: ... # omitting resource section from above
+
+  jobs:
+  - name: build-and-push
+   plan:
+   - get: concourse-examples
+  }}
+
+  \inline-header{Build the Image}
+
+  The second step in our job will build the container image.
+
+  To build the container image we are going to use the
+  \link{oci-build-task}{https://github.com/vito/oci-build-task}. The
+  \link{oci-build-task}{https://github.com/vito/oci-build-task} is a container
+  image that is meant to be used in a Concourse \reference{tasks}{task} to build other
+  container images. Check out the
+  \link{README}{https://github.com/vito/oci-build-task/blob/master/README.md} in
+  the repo for more details on how to configure and use the oci-build-task in
+  more complex build scenarios.
+
+  \titled-codeblock{build-push.yml}{yaml}{{
+  resources: ... # omitting resource section from above
+
+  jobs:
+  - name: build-and-push
+   plan:
```

Comment:
Ah looks like all the job configs at the `plan:` and step lines are one space unaligned in this file haha

![Screen Shot 2021-05-18 at 3 25 45 PM](https://user-images.githubusercontent.com/19290703/118711269-5ff5ab80-b7ed-11eb-9bfb-8f2ce23f75fc.png)